releases/release-1.29/release-notes/maps/pr-118568-map.yaml

```diff
@@ -1,3 +1,3 @@
 pr: 118568
 releasenote:
-  text: '`kubelet` exposes latency metrics of different stages of the node startup.'
+  text: '`kubelet` exposed latency metrics of different stages of the node startup.'
```

Comment:
i believe it should instead be: 

text: '`kubelet` will now expose latency metrics of different stages of the node startup.'